src/creds/VerifiableCredential.js

```diff
@@ -741,19 +741,38 @@ VerifiableCredentialBaseConstructor.CREDENTIAL_META_FIELDS = CREDENTIAL_META_FIE
 VerifiableCredentialBaseConstructor.getCredentialMeta = getCredentialMeta;
 VerifiableCredentialBaseConstructor.isMatchCredentialMeta = isMatchCredentialMeta;
 
+VerifiableCredentialBaseConstructor.create = async (identifier, issuer, expiryIn, ucas, version, evidence,
+  signerVerifier = null) => {
+  await schemaLoader.loadSchemaFromTitle(identifier);
```

Comment:
add comments about what is changing in terms of state

Reply:
I didn't get this, so we can't do new VC, only VC.create now?

Reply:
Done

Reply:
@dmelosantos yeah - because the schema's are loaded on the fly, the process has to be sync (e.g. the schema's could be loaded via http)